### DISCUSSION TITLE
Mouseover and mouseout firing repeatedly on links

### DISCUSSION THREAD

**Discussion by ZalgirisKaunas** [clientIO/joint#2765] [Q&A]
### Introduction

So when I hover on link transparent wrapper link tools appear fine and the event is fired only once. Still, if I hover further on the visual link's actual path even though it has pointerevents=none, it starts repeatedly firing mouseover and mouseout events. 

 vue.paper.on('link:mouseover', (linkView) => {
        console.log('mouse over link');
        console.log('linkView', linkView);
        vue.addTools(linkView);
      });

      vue.paper.on('link:mouseout', (linkView) => {
        console.log('mouse leave link');
        console.log('linkView', linkView);
        linkView.removeTools();
      });

![image](https://github.com/user-attachments/assets/8ef20b44-485e-474c-a732-359da8b52ea1)

addTools() contain      
const toolsView = new joint.dia.ToolsView({
        tools: [verticesTool, segmentsTool, removeButton],
 });
 linkView.addTools(toolsView);

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
Is there any reason why you don't use the `link:mouseenter` and `link:mouseleave` events?
